packages/polaris-viz/src/components/FunnelChartNext/Chart.tsx

```diff
@@ -286,7 +287,7 @@ export function Chart({
       return yPosition;
     }
 
-    return yPosition - TOOLTIP_HEIGHT;
+    return yPosition - TOOLTIP_HEIGHT - TOOLTIP_BOTTOM_OFFSET;
```

Comment:
Changing the line height tooltip rows slightly increased the size of the tooltip, however looking at the designs they are actually more inline now then they were before. I added this offset to make sure there is some space between the tooltip position and bar